slobrok/src/vespa/slobrok/sbmirror.cpp

```diff
@@ -256,10 +263,12 @@ MirrorAPI::handleReconnect()
             }
             double delay = _backOff.get();
             reSched(delay);
+            std::string cps = _slobrokSpecs.logString();
+            static const char * const msgfmt = "no location brokers available, retrying: %s (in %.1f seconds)";
```

Comment:
What is the rationale behind having this as `static` versus just being a regular const var? (also applies to usage in `sbregister.cpp`)

Reply:
> What is the rationale behind having this as static versus just being a regular const var?
I was worried about the compiler wanting to check the format string, but you're right it doesn't need to be static.